CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 
 include(Common)
 if(NOT MSVC) # cppnetlib does not build on Windows
-  git_subproject(RESTBridge https://github.com/BlueBrain/RESTBridge.git 4f16cfb)
+  git_subproject(RESTBridge https://github.com/BlueBrain/RESTBridge.git summit)
```

Comment:
Having this (summit) in master branch seems wrong to me. We are doing temporary hardcodings for summit.